### PR TITLE
Croatian support. 

### PR DESCRIPTION
**Pull request by MilanSCR**
This is what I've done so far translating to Croatian. I am working alone, some words simply can't be translated to make sense but I have tried my best so far in most cases.
I plan on doing the whole thing even though I'm really busy with IRL stuff but should be done over the weekend.

Discord: milan#7793 (banned from KB)
Krunker IGN: MilanSCR (tagged)
I am kind of exiled from the community and when it comes to being paid or like any compensation I see why I shouldn't get it ofc, I just wanted to help and this is the best I can do for Krunker.

### CODE REVIEW

**Comment by MilanSCR** [comment]
Translated as much as possible. Will be going back to polish and check for some stupid mistakes but the main thing is pretty much done.

**Comment by MilanSCR** [comment]
Translated as much as possible. Will be going back to polish and check for some stupid mistakes but the main thing is pretty much done.

**Comment by MilanSCR** [comment]
Think I did a good job, translated everything possible to make the most sense and work the best.

**Comment by mty22** [comment]
@MilanSCR nice!

can you please re-add `en.js` and push another commit ?

https://github.com/mty22/krunker.io-translations/blob/main/en.js

File: en.js [hunk 1]
```diff
@@ -1,747 +0,0 @@
-module.exports = `
```
Comment (mty22) [inline en.js:1]: need to keep this file, see other comment in PR please

**Review by mty22 — CHANGES_REQUESTED** [review]

**Comment by MilanSCR** [comment]
Closing because I'm stupid.
Opening new and actual working one.

**Pull request closed without merging**